.mailmap

```diff
@@ -0,0 +1,15 @@
+# Mailmap to correct users
+
+Joseph K. Bradley <joseph.kurata.bradley@gmail.com> jkbradley <joseph.kurata.bradley@gmail.com>
+Xiangrui Meng <meng@databricks.com>
```

Comment:
It seems that we don't need this line because I didn't use other names or emails. Same for L5, L10, L11, L12.